Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.Domain/Model/DbContextBase.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System.IO;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+
+namespace MvvmSeed.Domain.Model
+{
+    public abstract class DbContextBase : DbContext
```

Comment:
Yep, I thought the same while coding this.. 

> you are the single responsibility principle dude.

Hehe, yep, but I'm also the KISS principle dude when the degree of certainty is not enough to decide what approach to take. In such case, I prefer to write simple code and let the class evolve naturally.

The thing that bothers me most of this class is that I'm taking two different approaches for the same problem. Both Sqlite and InMemory are separate packages and added as extension methods to `DbContextOptionsBuilder`. I'm assuming Sqlite usage on Domain and letting UT project handle InMemory mode by itself.

Since the code is extremely simple, it's not a big deal right now, but I'll probably decide for one or the other soon, in a separate PR.